core/src/main/java/nl/nn/adapterframework/jms/JNDIBase.java

```diff
@@ -60,6 +65,13 @@
 
 	private Context context = null;
 
+	@Override
+	public void configure() throws ConfigurationException {
+		if(StringUtils.isEmpty(jndiContextPrefix)) {
+			jndiContextPrefix = AppConstants.getInstance(configurationClassLoader).getString("jndiContextPrefix","");
```

Comment:
I don't understand why this is necessary

Reply:
When a database is not initialized through Spring you manually have to set the jndiContextPrefix.. Maybe good to add as comment?